### PR TITLE
Bulk actions in datasets and maps revised and fixed

### PR DESCRIPTION
**Pull request by alejandraarri**
We have revised the conditions that make options appear or appear as disabled in the bulk actions menu. In short, actions that appear should be related to all selected visualizations, and if there is any shared visualization selected that prevents any action, this should appear as disabled.

### Acceptance
Make sure these are the actions that appear when selecting several visualizations.
https://docs.google.com/spreadsheets/d/1aXLj6CqqsF4ESqjFa-F0kDzttgO8x_WIKrqA1K6y9XE/edit#gid=0

### Related Issue
https://github.com/CartoDB/product/issues/228
https://github.com/CartoDB/product/issues/256

### CODE REVIEW

**Comment by alejandraarri** [comment]
Retest this please

File: lib/assets/javascripts/new-dashboard/styles/_utilities.scss [hunk 1]
```diff
@@ -27,3 +27,9 @@
     text-decoration: none;
   }
 }
+
+
+.u-is-disabled {
+  opacity: .5;
+  pointer-events: none;
+}
```
Comment (jesusbotella) [inline lib/assets/javascripts/new-dashboard/styles/_utilities.scss]: Let's put a newline here :)

Comment (alejandraarri) [reply lib/assets/javascripts/new-dashboard/styles/_utilities.scss]: Done! 👍

**Comment by alejandraarri** [comment]
Retest this please

**Comment by alejandraarri** [comment]
Retest this please

**Pull request merged**